lib/src/main/kotlin/pl/allegro/logic/operators/string/SubstringOperation.kt

```diff
@@ -0,0 +1,47 @@
+package pl.allegro.logic.operators.string
+
+import pl.allegro.logic.ClientLogic.isEmpty
+import pl.allegro.logic.ClientLogic.isEqual
+import pl.allegro.logic.ClientLogic.isNotEmpty
+import pl.allegro.logic.ClientLogicElement
+import pl.allegro.logic.ClientLogicOperator
+import pl.allegro.logic.ClientRegistryDataElement
+import pl.allegro.logic.NumberElement
+import pl.allegro.logic.ClientLogicMarker
+
```

Comment:
doubled empty line

Reply:
wdyt about adding detekt rule for such cases?

Reply:
rule added but not working idk why, i was looking for answers both internet and slack channels